updating/index.adoc

```diff
@@ -88,6 +88,13 @@ xref:../updating/updating-hardware-on-nodes-running-on-vsphere.adoc#updating-har
 * xref:../updating/updating-hardware-on-nodes-running-on-vsphere.adoc#updating-virtual-hardware-on-vsphere_updating-hardware-on-nodes-running-in-vsphere[Updating virtual hardware on vSphere]
 * xref:../updating/updating-hardware-on-nodes-running-on-vsphere.adoc#scheduling-virtual-hardware-update-on-vsphere_updating-hardware-on-nodes-running-in-vsphere[Scheduling an update for virtual hardware on vSphere]
 
+[id="updating-clusters-overview-sro"]
+== Updating a cluster that includes the SRO
```

Comment:
same comment as above: updating to upgrading

Reply:
guideline
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update says "update" is the preferred term.